addons/jest/README.md

```diff
@@ -67,125 +72,175 @@ You could create a `prebuild:storybook` npm script, which will never fail by app
 }
 ```
 
-### Register
+## Usage
 
-within `.storybook/main.js`:
+Assuming that you have already created a test file for your component (e.g., `MyComponent.test.js`).
 
-```js
-module.exports = {
-  addons: ['@storybook/addon-jest'],
-};
-```
+### Story-level
 
-## Usage
+In your story file, add a [decorator](https://storybook.js.org/docs/react/writing-stories/decorators) to your story's default export to display the results:
 
-Assuming that you have created test files `MyComponent.test.js` and `MyOtherComponent.test.js`
+```js
+// MyComponent.stories.js |  MyComponent.stories.jsx
 
-In your `story.js`
+import MyComponent from './MyComponent';
 
-```js
 import results from '../.jest-test-results.json';
+
 import { withTests } from '@storybook/addon-jest';
 
 export default {
+ component: MyComponent,
   title: 'MyComponent',
   decorators: [withTests({ results })],
 };
+```
+
+You can also add multiple tests results within your story by including the `jest` [parameter](https://storybook.js.org/docs/react/writing-stories/parameters), for example:
+
+```js
+// MyComponent.stories.js |  MyComponent.stories.jsx
+
+import MyComponent from './MyComponent'; 
+
+import results from '../.jest-test-results.json'; 
+
+import { withTests } from '@storybook/addon-jest'; 
 
-export const defaultView = () => <div>Jest results in storybook</div>;
-defaultView.parameters = {
-  jest: ['MyComponent.test.js', 'MyOtherComponent.test.js'],
+export default { 
+  component: MyComponent, 
+  title: 'MyComponent', 
+  decorators: [withTests({ results })],
+}; 
+
+const Template = (args) => <MyComponent {....args} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  text: 'Jest results in Storybook',
+};
+Default.parameters = {
+  jest: ['MyComponent.test.js', 'MyOtherComponent.test.js']
```

Comment:
Are we still using the template format? Or do we want to update this to CSF 3.0?

Reply:
@darleendenno I'm inclined to leave it as `template.bind({})` for the time being as CSF 3.0 is still being worked on and constantly mutating. And don't want to introduce misconceptions to new and old users.